modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/batch/04-object-definition.batch-engine-data.json

```diff
@@ -157,13 +129,13 @@
 				{
 					"DBType": "String",
 					"businessType": "Aggregation",
-					"externalReferenceCode": "RETURN_CHANNEL_TOTAL_AMOUNT",
+					"externalReferenceCode": "RETURN_CHANNEL_TOTAL_AUTHORIZED",
```

Comment:
Mmm... we are summarizing different things (liter authorized + meters authorized). Are we sure?

Reply:
why RETURN_CHANNEL_TOTAL_AUTHORIZED? I mean... why CHANNEL?

Reply:
they are names that if I remember correctly you actually chose them